docs/site/todo-tutorial-repository.md

```diff
@@ -24,67 +24,29 @@ For more information about Repositories, see
 
 ### Create your repository
```

Comment:
We should explain what a repository is and why it is required. LB3 users not familiar with this concept will be left  wondering about the "added complexity".

Reply:
isn't it explained already here https://loopback.io/doc/en/lb4/Repositories.html ?

Reply:
@hacksparrow, there is a section above the `Create your repository` that explains what a repository is.  What I've removed is the details that what the index.ts used for.